android/anc/src/main/java/org/smartregister/fhircore/anc/ui/madx/details/NonAncPatientItemMapper.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.anc.ui.madx.details
+
+import com.google.android.fhir.logicalId
+import org.hl7.fhir.r4.model.Patient
+import org.smartregister.fhircore.anc.AncApplication
+import org.smartregister.fhircore.anc.data.madx.model.AncPatientItem
+import org.smartregister.fhircore.engine.data.domain.util.DomainMapper
+import org.smartregister.fhircore.engine.util.extension.atRisk
+import org.smartregister.fhircore.engine.util.extension.extractAge
+import org.smartregister.fhircore.engine.util.extension.extractGender
+import org.smartregister.fhircore.engine.util.extension.extractName
+
+object NonAncPatientItemMapper : DomainMapper<Patient, AncPatientItem> {
```

Comment:
calling this from `ToAncPatientIemMapper` is probably more descriptive of what this does.

Although, it might be better to make this generic as a `PatientItemMapper` in engine, then subclass (or implement, not sure if an interface is better) it here as a `ToAncPatientItemMapper`